src/Tizen.TV.UIControls.Forms/FocusFrame.cs

```diff
@@ -82,12 +186,41 @@ protected override void OnPropertyChanged([CallerMemberName] string propertyName
         /// <param name="isFocused">This parameter indicates whether the content is focused.</param>
         protected virtual void OnContentFocused(bool isFocused)
         {
-            BackgroundColor = isFocused ? FocusedColor : Color.Transparent;
+            BackgroundColor = isFocused ? FocusedColor : UnfocusedColor;
```

Comment:
I think UnfocusedColor should be set to BackgroundColor when initializing FocusFrame or setting UnfocusedColor value.

Reply:
Yes, I update `FocusedColor` and `UnfocusedColor` `propertyChanged` handler